tutorials/promscale-install.md

```diff
@@ -0,0 +1,146 @@
+## 3. Up and Running: Install Prometheus, Promscale and TimescaleDB [](up-and-running)
+
+We recommend four methods to setup and install Promscale:
+1. [TOBS - The Observability Stack for Kubernetes ][tobs-github] (recommended for kubernetes deployments)
+2. Docker (instructions detailed below) 
+3. [Helm][promscale-helm-chart]
+4. [Bare Metal][promscale-baremetal-docs]
+
+> :TIP: See the [Promscale github installation guide][promscale-github-installation] for more information on installation options.
+
+For demonstration purposes, we will use Docker to get up and running with Promscale.
+
+The easiest way to get started is by using Docker images. Make sure you have Docker installed on your local machine ([Docker installation instructions][docker]).
+
+The instructions below have 4 steps:
+1. [Install TimescaleDB](#install-timescaledb)
+2. [Install Promscale](#install-promscale)
+3. [Install node_exporter](#install-node-exporter)
+4. [Install Prometheus](#install-prometheus)
+
+>:WARNING: The instructions below are local testing purposes only and should not be used to set up a production environment.
+
+### Step 1: Install TimescaleDB [](install-timescaledb)
+
+First, let's create a network specific to Promscale and TimescaleDB:
+
+```bash
+docker network create --driver bridge promscale-timescaledb
+```
+
+
+Secondly, let's install and spin up an instance of TimescaleDB in a docker container.  This is where Promscale will store all metrics data scraped from Prometheus targets. 
+
+We will use a docker image which has the`promscale` PostgreSQL extension already pre-installed:
+
+```bash
+docker run --name timescaledb \
+    --network promscale-timescaledb \
+    -e POSTGRES_PASSWORD=<password> -d -p 5432:5432 \
+    timescaledev/timescaledb-ha:pg12-latest
+```
+The above commands creates a TimescaleDB instanced named `timescaledb` (via the `--name` flag), on the network named `promscale-timescale` (via the `--network` flag), whose container will run in the background with the container ID printed after created (via the `-d` flag), with port-forwarding it to port `5432` on your machine (via the `-p` flag).
+
+> :WARNING: Note we set the `POSTGRES_PASSWORD` environment variable (using the `-e` flag) in the command above. Please ensure to replace `<password>` with the password of your choice for the `postgres` superuser.
+> 
+> Note that for production deployments, you will want to fix the docker tag to a particular version instead of `pg12-latest`
+
+### Step 2: Install Promscale [](install-promscale)
+
+Since we have TimescaleDB up and running, let’s spin up a [Promscale][promscale-github], using the [Promscale docker image][promscale-docker-image] available on Docker Hub:
```

Comment:
> let’s spin up a [Promscale][promscale-github]

"spin up Promscale" or "spin up a Promscale instance"?

Reply:
changed to "Promscale instance"